### DISCUSSION TITLE
SWL and runup

### DISCUSSION THREAD

**Discussion by davefavismortlock** [coastalme/coastalme#4] [Ideas]
OK all: so CoastalME now uses mean SWL to determine the start elevation of beach Dean profiles. The question is: should we also use runup? 

Manuel said "Regarding the problem with the wave run-up, I am in complete agreement with you [@Dave Favis-Mortlock] on leaving it as it is. From my point of view, the wave run-up is a process of water ascent along the beach profile, which can locally increase the flood elevation (in time and space) but does not significantly modify the profile shape, so I would not consider it. Additionally, the Dean profile should establish a “hypothetical equilibrium state to which the beach tends”, something medium to long-term, not short-term like the wave run-up. CShore calculates it based on empirical formulas, so I prepared the CoastalME code to calculate it using different empirical formulas without needing to extract it from CShore."

So we don't consider runup for beach Dean profiles, I guess. Comments, anyone?

Manuel, where in CoastalME do we use runup now? Where would we like to use it in future? Please excuse me if these are stupid questions :-)

**Comment by mcobosb** [comment]
Run-up is obtaining from empirical formulas and written into an output file. It would be useful if we will compute the flooding coastline due to all (surge, setup, mean, astronomical, run-up, ...) components.

**Reply by davefavismortlock** [reply]
OK, understood. So what do we then do (within CoastalME) with this "flooding coastline due to all (surge, setup, mean, astronomical, run-up, ...) components"?

**Reply by mcobosb** [reply]
I made changes on the code to obtain that flooding coastline but I'm not sure if they were appended into the last version due to those results were interesting for Andalusian project but likely not to the main CoastalME development.

**Reply by davefavismortlock** [reply]
OK Manuel thanks. What do I look for in the CME code to find if these changes are included? Any clue will be helpful :-)

**Reply by MikeWalkden** [reply]
I'll just flag that in SCAPE is seemed beneficial use runup to define the landward limit of the distribution of shore platform erosion. This seemed to give a better result for cliff toe level. This is something for me to keep in mind when I review the CME platform erosion code.

**Comment by MikeWalkden** [comment]
I agree on this - the origin of the Dean profile should not be based on calculated runup
